Neos.ContentRepository/Classes/Command/NodeCommandController.php

```diff
@@ -118,6 +119,90 @@ public function repairCommand($nodeType = null, $workspace = 'live', $dryRun = f
         $this->outputLine('Node repair finished.');
     }
 
+    /**
+     * Run a status report of nodes
+     *
+     * {pluginDescriptions}
+     * <b>Examples:</b>
+     *
+     * ./flow node:status
+     *
+     * ./flow node:status --node-type Neos.NodeTypes:Page
+     *
+     * ./flow node:status --workspace user-robert --only removeOrphanNodes,removeNodesWithInvalidDimensions
+     *
+     * ./flow node:status --skip removeUndefinedProperties
+     *
+     * @param string $nodeType Node type name, if empty update all declared node types
+     * @param string $workspace Workspace name, default is 'live'
+     * @param boolean $dryRun Don't do anything, but report actions
+     * @param boolean $cleanup If false, cleanup tasks are skipped
+     * @param string $skip Skip the given check or checks (comma separated)
+     * @param string $only Only execute the given check or checks (comma separated)
+     * @return void
+     */
+    public function statusCommand($nodeType = null, $workspace = 'live', $cleanup = true, $skip = null, $only = null)
+    {
+        $dryRun = true;
+        $summary = true;
+
+        $this->outputLine('Run status report..');
+
+        $this->pluginConfigurations = self::detectPlugins($this->objectManager);
+
+        if ($this->workspaceRepository->countByName($workspace) === 0) {
+            $this->outputLine('Workspace "%s" does not exist', [$workspace]);
+            exit(1);
```

Comment:
```
$this->quit(1);
return;
```
would be the "proper" way - exit prevents the framework from being shut down correctly